ccmlib/scylla_node.py

```diff
@@ -316,6 +316,8 @@ def _start_scylla(self, args, marks, update_pid, wait_other_notice,
                 raise NodeError(f"Error starting node {self.name}",
                                 self._process_scylla)
 
+        self.node_hostid = None
```

Comment:
Nitpick: Would have been useful to get the explanation in the commit message as a comment, not (just) as a commit message.

Reply:
added in next version